index.js

```diff
@@ -176,6 +175,35 @@ class WebSocketJSONServer extends EventEmitter {
     })
   }
 
+
+  // payload format: { type: 'messageType', data: $someJSON }
+  sendJson(ws,payload) {
+    return this.send(ws,JSON.stringify(payload));
+  }
+
+
```

Comment:
broadcast is new but not hot. It just walks connections and sends them all a payload